src/ddprof_worker.cc

```diff
@@ -231,7 +232,7 @@ DDRes ddprof_unwind_sample(DDProfContext &ctx, perf_event_sample *sample,
   if (us->_dwfl_wrapper && us->_dwfl_wrapper->_inconsistent) {
     // Loaded modules were inconsistent, assume we should flush everything.
     LG_WRN("(Inconsistent DWFL/DSOs)%d - Free associated objects", us->pid);
-    DDRES_CHECK_FWD(worker_pid_free(ctx, us->pid));
```

Comment:
It weird to have to call `PerfClock::now()` here, perhaps having two versions of `worker_pid_free` (with and without timestamp) or having `PerfClock::time_point::max()` as default value for `timestamp` in `worker_pid_free` would be more natural.